### PR TITLE
Minor fix for AIX

### PR DESCRIPTION
**Pull request by jlcharton**
BTW, are all the includes really necessary?

### CODE REVIEW

File: src/OVAL/probes/SEAP/sch_pipe.c [hunk 1]
```diff
@@ -31,7 +31,9 @@
 #include <limits.h>
 #include <sys/stat.h>
 #include <sys/uio.h>
+#if !defined(_AIX)
 #include <sys/unistd.h>
```
Comment (mpreisler) [inline src/OVAL/probes/SEAP/sch_pipe.c]: are you sure this is not available on AIX?

Comment (jlcharton) [reply src/OVAL/probes/SEAP/sch_pipe.c]: On my AIX, I've got this compiler error: 1506-296 (S) #include file <sys/unistd.h> not found.
But beyond AIX, are you sure that sys/uio.h, sys/unistd.h and even fcntl.h are really necessary?
On my CentOS, Solaris and AIX boxes, I can compile sch_pipe.c without these 3 includes.

Comment (mpreisler) [reply src/OVAL/probes/SEAP/sch_pipe.c]: In that case just remove the include, it could be redundant/forgotten.

**Comment by jan-cerny** [comment]
@jlcharton I would prefer if those 2 commits are squashed into one commit and I would be happy if the commit message says what is removed and why.

**Comment by jan-cerny** [comment]
@jlcharton I know that working with commits caused you problems in past, so I have done that manually. See https://github.com/OpenSCAP/openscap/commit/fdcc3324d02c96e3b1c6f1a3bb7d29d6a0355602
I recommmend https://git-scm.com/book/en/v2/, it's a good read.

**Comment by jlcharton** [comment]
@jan-cerny: thank you :-)

**Pull request closed without merging**